README.md

```diff
@@ -31,47 +31,53 @@ Works with various Large Language Models from industry leaders such as Anthropic
 
 ## Branches 🌿
 
-- **`main`**: Stable branch, uses hosted services like Pinecone for vectors and Firebase for database and storage.
-- **`dev`**: Includes the latest features, fully functional with a self-hosted MongoDB for enhanced privacy, still uses Pinecone for vectors.
-- **`vector-db`**: Focuses on integrating various vector databases, both self-hosted and cloud-based, for ultimate privacy and control.
+- **`main`**: Stable branch, uses hosted services like Pinecone for vectors and Firebase for database and storage. This is what gets deployed in production
+- **`dev`**: Includes the latest features, fully functional with a self-hosted MongoDB for enhanced privacy, still uses Pinecone for vectors. This is deployed in staging.
 
 ## Setup
 
-To run the development server directly:
-
-- .env.local or .env.development must be present
-
+## Development
+### Source
+#### Installing dependencies
+You need NodeJS and pnpm to build and run the app from source.
+Install dependencies:
+```bash
+pnpm install
```

Comment:
we can just say `npm install` 
if the user has pnpm or yarn they can choose to use that. From our point of view it doesn't matter